package.json

```diff
@@ -15,6 +15,7 @@
     "@edx/frontend-enterprise-utils": "^4.1.0",
     "@edx/frontend-platform": "5.0.0",
     "@edx/paragon": "20.45.4",
+    "@loadable/component": "5.16.3",
```

Comment:
We anticipate using this package for future iterations on performance in the very near future. imported it preemptively in this PR